packages/launcher-app/.gitignore

```diff
@@ -27,4 +27,5 @@ yarn-error.log*
 .idea/
 *.swp
 .vscode/
-*.code-workspace
\ No newline at end of file
+*.code-workspace
+.history
```

Comment:
why does `launcher-app` has it's own `.gitignore`, the one in the root of the project should be enough?

Reply:
@edewit right that's weird ^^